docs/site-example/i18n/de.po

```diff
@@ -28,35 +28,63 @@ msgstr ""
 msgid "gluon-config-mode:pubkey"
 msgstr ""
 "<p>Dies ist der öffentliche Schlüssel deines Freifunk-Knotens. Erst nachdem "
-"er auf den Servern des Freifunk-Projektes auf Alpha Centauri eingetragen wurde, "
-"kann sich dein Knoten mit dem Mesh-VPN dort verbinden. Bitte "
-"schicke dazu diesen Schlüssel und den Namen deines Knotens "
+"er auf den Servern des Freifunk-Projektes auf Alpha Centauri eingetragen "
+"wurde, kann sich dein Knoten mit dem Mesh-VPN dort verbinden. Bitte schicke "
+"dazu diesen Schlüssel und den Namen deines Knotens "
 "(<em><%=pcdata(hostname)%></em>) an "
-"<a href=\"mailto:keys@alpha-centauri.freifunk.net?subject=<%= urlencode('Anmeldung: ' .. hostname) %>&amp;"
-"body=<%= urlencode('# ' .. hostname .. '\n' .. pubkey) %>\">keys@alpha-centauri.freifunk.net</a>."
+"<a href=\"mailto:keys@alpha-centauri.freifunk.net?subject="
+"<%= urlencode('Anmeldung: ' .. hostname) %>&amp;body="
+"<%= urlencode('# ' .. hostname .. '\n# ' .. sysconfig.primary_mac .. '\nkey ') %>"
+"%22<%= pubkey %>%22;"
+"<%= urlencode('\n\nIch habe zur Kenntnis genommen, dass der im ') %>"
+"<%= urlencode('Knoten von mir eingetragene Kontakt im Internet ') %>"
+"<%= urlencode('öffentlich einsehbar ist und von beliebigen Diensten ') %>"
+"<%= urlencode('(z.B. der Freifunk-Karte) verwendet werden kann.') %>"
+"<%= urlencode('\n\nIch möchte informiert werden, falls mein Knoten ') %>"
+"<%= urlencode('offline ist.\nBitte meldet mich beim Knotenalarm an.') %>"
+"<%= urlencode('\n\nGruß, \n\n') %>"
+"\">keys@alpha-centauri.freifunk.net</a>. Deine E-Mail Adresse wird "
+"selbstverständlich vertraulich behandelt und nicht weitergegeben."
 "</p>"
 "<div class=\"the-key\">"
-" # <%= pcdata(hostname) %>"
-" <br />"
+"# <%= pcdata(hostname) %><br />"
 "<%= pubkey %>"
 "</div>"
+"<p>Dein Knoten startet gerade neu und wird anschließend versuchen, sich mit "
+"anderen Freifunkknoten in seiner Nähe und über deine Internetleitung, online "
```

Comment:
i think the following is sufficient: "und über deine Internetverbindung mit einem"